vulcan/formula.py

```diff
@@ -75,14 +77,19 @@ def install(self, force=False):
                 shutil.rmtree(destination)
 
             # Move formula into place.
+            log.info('Moving formula %s to %s' %(self.name, destination))
             origin = os.path.join(tmpdir, self.origin_name)
             shutil.move(origin, destination)
 
             # Save state file.
             with open(self.state_file, 'w') as fh:
                 json.dump(self.as_dict(), fh)
 
+        except Exception as e:
+            log.error('Could not get formula: %s , exception: %s' % (self.name, e))
+
         finally:
+            log.debug('Removing tmpdir %s' % tmpdir)
```

Comment:
Still incorrect format. Here's the doc on correct formatting. 

https://docs.python.org/3/howto/logging.html#logging-variable-data

The difference is cost. If you build the string prior to logging, you spend the cost for string creation even if nothing is logged. Doing it with `log.error('My name is %s', 'Seth')` doesn't build the string unless there's actually an error.

This should error when doing a `make lint` with the message "logging-not-lazy".